examples/experimental/dagster-airlift/README.md

```diff
@@ -273,31 +327,30 @@ The migration file acts as the source of truth for migration status. A task whic
 
 #### Migrating common operators
 
-For some common operator patterns, like our dbt operator, Dagster supplies factories to build software defined assets for our tasks:
+For some common operator patterns, like our dbt operator, Dagster supplies factories to build software defined assets for our tasks. In fact, the `dbt_defs` factory used earlier already backs its assets with definitions, so we can toggle the migration status of the `build_dbt_models` task to `migrated: True` in the migration state file:
 
-```python
-from dagster_airlift.core import build_defs_from_airflow_instance, dag_defs, task_defs
-from dagster_dbt import DbtProject
-
-defs = build_defs_from_airflow_instance(
-    airflow_instance=airflow_instance,
-    defs=dag_defs(
-        "rebuild_customers_list",
-        ...,
-        task_defs(
-            "build_dbt_models",
-            dbt_defs(
-                manifest=dbt_project_path() / "target" / "manifest.json",
-                project=DbtProject(dbt_project_path()),
-            ),
-        ),
-    )
-)
+```yaml
+tasks:
+  - id: load_raw_customers
+    migrated: False
+  - id: build_dbt_models
+    migrated: True
+  - id: export_customers
+    migrated: False
 ```
 
+You can now run the `rebuild_customers_list` DAG in Airflow, and the `build_dbt_models` task will be executed in a Dagster run.
+
+
+<p align="center">
+
+![dbt build executing in Dagster](./images/migrated_dag.png)
+
+</p>
+
 #### Migrating custom operators
 
-For all other operator types, we recommend creating a new factory class whose arguments match the inputs to your Airflow operator. Then, you can use this factory to build definitions for each Airflow task.
+For all other operator types, we recommend creating a new factory class whose arguments match the inputs to your Airflow operator. Then, you can use this factory to build asset definitions for each Airflow task.
```

Comment:
We no longer have this fully supported factory class thing, so we can just say a factory function.

Reply:
Also we can just say "build definitions" for each Airflow task